<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Instagram</class>
 <widget class="QDialog" name="Instagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/instagram.png</normaloff>:/images/instagram.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
}

QDialog{
background: green;
}

QLabel#labelTitulo{
color:white;
font-size:28px;
font-family:century gothic;
}

QLabel#label{
color:white;
font-size:20px;
font-family:monospace;
}

QLabel#labe_4l{
color:white;
font-size:20px;
font-family:monospace;
}

QLabel#label_5{
color:white;
font-size:20px;
font-family:sans-serif;
}

QLabel#label_2{
color:white;
font-size:20px;
font-family:sans-serif;
}

QLabel#labelFondo{
color:white;
font-size:12px;
font-family:monospace;
}



QLineEdit{
background: transparent;
border:none;
border-bottom: 2px solid white;
color:white;
font-size:16px;
text-align:center;
}

QPushButton#botonData{
background: #2d89ef;
color:white;
border-radius:15px;
font-size:16px;
}

QPushButton#botonData:hover{
font-size:18px;
background: #2b5797;
color:white;
border-radius:15px;
font-weight:bold;
}

QPushButton{
background: #fe8c00 ;
color:white;
border-radius:15px;
font-size:16px;
}

QPushButton:hover{
font-size:18px;
background:#f83600;
color:white;
border-radius:15px;
font-weight:bold;
}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Location</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>loc_id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Latitutde</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Longitude</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="botonData">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Scrape</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="posts_per_page_entry">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Posts per page:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="posts_per_file_entry">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Posts per file:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>921</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Instagram geo-tagged posts scraper</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFondo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonRestart">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>570</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue Scrape</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>End Cursor:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_cursor_entry">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>121</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/ig_geo.ico</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/background.png</pixmap>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>tableWidget</zorder>
  <zorder>botonData</zorder>
  <zorder>posts_per_page_entry</zorder>
  <zorder>label</zorder>
  <zorder>posts_per_file_entry</zorder>
  <zorder>label_2</zorder>
  <zorder>labelTitulo</zorder>
  <zorder>labelFondo</zorder>
  <zorder>botonRestart</zorder>
  <zorder>label_5</zorder>
  <zorder>end_cursor_entry</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
